lib/ckb/api.rb

```diff
@@ -7,7 +7,8 @@
 
 module Ckb
   URL = "http://localhost:8114"
-  DEFAULT_CONFIGURATION_FILENAME = File.expand_path("../../../conf.json", __FILE__)
+  MRUBY_CONFIGURATION_FILENAME = File.expand_path("../../../conf.json", __FILE__)
```

Comment:
I feel we can use one JSON file to save all the configurations.
One more thing, what if I want to update a binary multiple times, or update several different binary files. How do I use the configuration to load them separately? Plan that as a future feature?

Reply:
Basic wallet can use both system scripts and mruby scripts, so it must to know the context whether system scripts or mruby scripts using now. It can save to one file, but still should tell load function load system scripts as context or mruby scripts.

Reply:
In real world scenario, one can send a transaction having multiple outputs, each using a different script.

Although at first we might just only have one kind of type_hash as default addressing mechanism but it's not harming to consider that a little bit, even if we don't do that now.

Reply:
I'll merge conf to one file first.